src/bci_build/package/appcontainers.py

```diff
@@ -203,87 +203,96 @@ def _envsubst_pkg_name(os_version: OsVersion) -> str:
     + [(pg_ver, OsVersion.TUMBLEWEED) for pg_ver in (14, 13, 12)]
 ]
 
-PROMETHEUS_PACKAGE_NAME = "golang-github-prometheus-prometheus"
+_PROMETHEUS_PACKAGE_NAME = "golang-github-prometheus-prometheus"
 PROMETHEUS_CONTAINERS = [
     ApplicationStackContainer(
         package_name="prometheus-image",
         os_version=os_version,
         is_latest=os_version in CAN_BE_LATEST_OS_VERSION,
         name="prometheus",
         pretty_name="Prometheus",
-        package_list=[PROMETHEUS_PACKAGE_NAME],
-        version="%%prometheus_version%%",
+        package_list=[_PROMETHEUS_PACKAGE_NAME],
+        version="%%prometheus_patch_version%%",
+        additional_versions=[
+            "%%prometheus_minor_version%%",
+            "%%prometheus_major_version%%",
+        ],
         version_in_uid=False,
         entrypoint=["/usr/bin/prometheus"],
         replacements_via_service=[
             Replacement(
-                regex_in_build_description="%%prometheus_version%%",
-                package_name=PROMETHEUS_PACKAGE_NAME,
-                parse_version="patch",
+                regex_in_build_description=f"%%prometheus_{level}_version%%",
+                package_name=_PROMETHEUS_PACKAGE_NAME,
+                parse_version=level,
             )
+            for level in ("major", "minor", "patch")
```

Comment:
maybe consider creating an Enum for levels since it is used in multiple places.

Reply:
Right, that's actually better